lib/ansible/modules/messaging/rabbitmq_user.py

```diff
@@ -229,6 +229,14 @@ def has_tags_modifications(self):
     def has_permissions_modifications(self):
         return sorted(self._permissions) != sorted(self.permissions)
 
+    def has_password_modification(self):
```

Comment:
I reviewed the change again and I think a check for a none password is missing, because the password field is optional we should only validate the password if not None. 

~~~yaml
if self.password is None:
  return False
~~~

Reply:
@resmo Great catch! I'll change the PR. ;-)

Reply:
@resmo Fixed in 9831ab7.